h2o-extensions/xgboost/src/test/java/hex/tree/xgboost/XGBoostTest.java

```diff
@@ -294,6 +362,9 @@ public void WeatherBinary() {
       model = new hex.tree.xgboost.XGBoost(parms).trainModel().get();
       Log.info(model);
 
+      FrameMetadata metadataAfter = new FrameMetadata(tfr);
+      // TODO: this fails; put back: Assert.assertEquals(metadataBefore, metadataAfter);
```

Comment:
Does this still fail?

Reply:
Yes, I found a previously unknown bug via this test.  I commented out the assert because it will fail until the bug is fixed.  See: https://0xdata.atlassian.net/browse/PUBDEV-5222